src/main/java/stsjorbsmod/actions/HealOnFatalAttackAction.java

```diff
@@ -0,0 +1,39 @@
+package stsjorbsmod.actions;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.HealAction;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.powers.MinionPower;
+import com.megacrit.cardcrawl.vfx.combat.FlashAtkImgEffect;
+
+public class HealOnFatalAttackAction extends AbstractGameAction {
```

Comment:
I think `Damage` instead of `Attack` would be more in line with similar action names (`VampireDamageAction`, `DamageAction`, etc)